examples/eth_bots/main.py

```diff
@@ -114,6 +120,32 @@ def main():  # TODO: Move much of this out of main
                 # TODO: deliver crash report
 
 
+def get_wait_for_new_block(web3: Web3) -> bool:
```

Comment:
Awesome! Can you clear the TODO on line 113? `# TODO: if provider.auto_mine is set then run the mine function`